modules/programs/rbw.nix

```
@@ -87,7 +81,7 @@ in {
         {
           email = "name@example.com";
           lock_timeout = 300;
-          pinentry = "gnome3";
+          pinentry = "pkgs.pinentry-gnome";
```

I guess this should be
```suggestion
          pinentry = pkgs.pinentry-gnome;
```